packager/react-packager/src/DependencyResolver/DependencyGraph/ResolutionRequest.js

```diff
@@ -336,6 +336,8 @@ class ResolutionRequest {
         file = potentialModulePath + '.native.js';
```

Comment:
As much as I'd love to see `.jsx` files here you have a good point on supporting other extensions and the fragmentation it could lead too, I didn't see it that way. Thanks @vjeux